lib/hydra/derivatives/processors/document.rb

```diff
@@ -3,7 +3,7 @@ class Document < Processor
     include ShellBasedProcessor
 
     def self.encode(path, format, outdir)
-      execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} #{path}"
+      execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} \"#{path}\""
```

Comment:
Should we use `Shellwords.escape()` here?  This would handle files that include a quote.